README.md

```diff
@@ -0,0 +1,63 @@
+# Green Jobs
+
+## Identifying Green Jobs 
+
+The aim of this project is to identify green jobs within the [Open Jobs Observatory](https://github.com/nestauk/ojd_daps) job adverts database as part of a case study deliverable for the Department of Education. 
+
+This repo contains the current methodology for doing so. At a high level, the revised, supervised methodology is as follows:
+
+1) Preprocess the text to: 
+	a) Remove punctuation
+	b) Detect sentences
+	c) Lemmatise terms
+	d) lowercase terms
+	e) remove numbers
+	f) remove stopwords
+
+2) Generate bespoke normalised green count feature based on keyword expansion   
+3) Create tfidf vectors and stack bespoke feature 
+4) Oversample labelled training embeddings to address class imbalance using a Synthetic Minority Oversampling Technique
+5) Train a gradient boosted decision tree algorithm to predict whether jobs are green or not_green      
+The final output is a dictionary of jobs that have been classified as 'green' following the above methodology. 
```

Comment:
There's quite a few places throughout the code where "a dictionary of jobs that have been classified as 'green'" is mentioned, but without some additional documentation or an example it's hard to know exactly what the structure of this is, whether it is consistent throughout the codebase and what behaviour to expect from a couple of the functions.